dump/plugins/executor.py

```diff
@@ -24,3 +25,14 @@ def execute(self, params):
     @abstractmethod
     def get_all_args(self, ns):
         pass
+    
+    def add_to_ret_template(self, table, db, keys, err, add_to_tables_not_found=True):
+        if db not in self.ret_temp:
+            return []
+
+        if not err and keys:
+            self.ret_temp[db]["keys"].extend(keys)
+            return keys
```

Comment:
The return value is different from what was there in the older add_to_ret_template.
Is there any reason for the change? This return value is not being used anywhere.

Reply:
Previously, different modules has different implementations of the same function. Route module was using the keys returned by this function to do further processing. The other modules were just using this as a boolean return.

 An empty list is Falsy, so this approach is more general